[Alerting][Docs] Changed alerting documentation to point to a single source of explaining the configurations. (#92942)
elastic/kibana: docs/user/alerting/defining-alerts.asciidoc

[NOTE]
==============================================

> Actions are not required on alerts. In some cases you may want to run an alert without actions first to understand its behavior, and <<action-settings, configure actions>> later.

Reviewer: ```suggestion
Actions are not required on alerts. You can run an alert without actions to understand its behavior, and then <<action-settings, configure actions>> later.
```
